cloudygames/views.py

```diff
@@ -1,28 +1,99 @@
 from django.shortcuts import render
+from django.core import serializers
 
-from rest_framework import viewsets, generics, status
+from rest_framework import viewsets, generics, status, filters
 from rest_framework.decorators import api_view
 from rest_framework.response import Response
 
 from cloudygames.serializers import GameSerializer, GameSessionSerializer, PlayerSaveDataSerializer
 from cloudygames.models import Game, GameSession, PlayerSaveData
 
+import django_filters
+import json
+
+
+######################## Filter ############################
+
+
+class GameFilter(django_filters.FilterSet):
```

Comment:
checkout some [usage](http://django-filter.readthedocs.org/en/latest/usage.html) guide, you should be able to figure out how to write a custom filter for `is_owner` here. Check the BooleanFilter or MethodFilter, those 2 may be relevant or helpful class.